galaxy/Dockerfile

```diff
@@ -19,7 +19,7 @@ GALAXY_CONFIG_DIR=/etc/galaxy
 ENV GALAXY_CONFIG_FILE=$GALAXY_CONFIG_DIR/galaxy.ini \
 GALAXY_CONFIG_JOB_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_conf.xml \
 GALAXY_CONFIG_JOB_METRICS_CONFIG_FILE=$GALAXY_CONFIG_DIR/job_metrics_conf.xml \
-GALAXY_VIRTUALENV=$GALAXY_ROOT/.venv/ \
+GALAXY_VIRTUAL_ENV=/galaxy_venv \
```

Comment:
Correct. Currently the .venv isn't exposed outside of the docker container. Since jobs on nodes run as the `galaxy` user, who can's create docker instances for things to run inside (not that I'd support the added overhead of doing that anyway), a lot of tools will break unless `.venv` is copied elsewhere. I did that this morning on our internal instance and that stopped the steady stream of bug report emails I've been getting since switching everything over to the cluster on Monday.